### PR TITLE
CI: add ClamAV scan job

### PR DESCRIPTION
**Pull request by nurmukhametov**
It adds a job to scan trunk archives with ClamAV.

This partially addresses #2913.

This PR also fixes appveyor build for Linux that was broken after https://github.com/ispc/ispc/pull/2894

### CODE REVIEW

File: .github/workflows/clamav.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+name: Scan with ClamAV
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every day at 22:00 UTC
+    - cron: '0 22 * * *'
+
+env:
+  zip_url: https://ci.appveyor.com/api/projects/ispc/ispc/artifacts/build%2Fispc-trunk-windows.zip?job=Environment%3A%20APPVEYOR_BUILD_WORKER_IMAGE%3DVisual%20Studio%202019%2C%20LLVM_VERSION%3Dlatest
```
Comment (aneshlya) [inline .github/workflows/clamav.yml:13]: What do you think about scanning the last Github release (which we need for release security scans)?

File: .github/workflows/clamav-release.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        TAR="ispc-examples-v$V.tar.gz ispc-v$V-linux-oneapi.tar.gz \
+             ispc-v$V-linux.aarch64.tar.gz ispc-v$V-linux.tar.gz \
+             ispc-v$V-macOS.arm64.tar.gz ispc-v$V-macOS.universal.tar.gz \
+             ispc-v$V-macOS.x86_64.tar.gz"
+        ZIP="ispc-examples-v$V.zip ispc-v$V-windows.zip"
+        for f in $TAR $ZIP; do
+          wget -O $f ${{ env.release_base_url }}/v$V/$f
+        done
+        mkdir zip tar
+        for f in $ZIP; do
```
Comment (aneshlya) [inline .github/workflows/clamav-release.yml:51]: It looks like that ClamAV can scan archives `--scan-archive --archive-verbose` so probably no need to unpack them?
https://manpages.ubuntu.com/manpages/jammy/man1/clamscan.1.html

Comment (nurmukhametov) [reply .github/workflows/clamav-release.yml]: I have added it, but it doesn't seem to be reliable enough for me. It doesn't scan all archives by some reason ([link](https://github.com/nurmukhametov/ispc/actions/runs/10216148461)).

Comment (aneshlya) [reply .github/workflows/clamav-release.yml]: OK

**Review by aneshlya — APPROVED** [review]
Looks good!

**Pull request merged**